srcpkgs/hunspell-el_GR/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'hunspell-el_GR'
+pkgname=hunspell-el_GR
+version=0.4.0
+revision=1
+archs=noarch
+create_wrksrc=yes
+hostmakedepends="unzip"
+short_desc="Greek el_GR dictionary for hunspell"
+maintainer="Nicholas Christopoulos <nereus@freemail.gr>"
+license="GPL-2.0"
```

Comment:
`srcpkgs/hunspell-el_GR/template:10: use SPDX id for 'GPL-2.0' license or see Manual.md`
You miss the `-only` or `-or-later` suffix, please add whatever the case is here. Also squash all commit into one.

Reply:
well its not clear since is a mess the licenses of orthos... the package(?) is under BSD 3-clause, but the data inside the tar.gz are under GPL. It is not clear if it is GPL2 or later since the specific file for the specific data files was written at 2002! GPL-2.0-only is good for now until they noted.

Reply:
Since there is nothing else specified it should be `-only` Please squash everything into a single commit and this should be good to go.